core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/SampleAnalyteImpl.java

```diff
@@ -46,40 +31,6 @@
   private String region;
```

Comment:
Region is to be moved into SampleTissue (not sure if that's relevant to this PR).